data-model/virtualcollections/v.0.1.0/schemas/virtualcollections.json

```diff
@@ -1 +1,115 @@
-
+{
+  "$schema": "http://json-schema.org/draft-2020-12/schema#",
+  "type": "object",
+  "properties": {
+    "ods:id": {
+      "type": "string",
+      "pattern": "^https:\\/\\/hdl\\.handle\\.net\\/20\\.5000\\.1025\\/.{3}-.{3}-.{3}$",
+      "examples": ["https://hdl.handle.net/20.5000.1025/XXX-XXX-XXX"],
+      "description": "Unique identifier for the object."
+    },
+    "ods:type": {
+      "type": "string",
+      "examples": ["https://doi.org/21.T11148/2037de437c80264ccbce"],
+      "description": "This is the handle of the type of the object."
+    },
+    "ods:version": {
+      "type": "integer",
+      "examples": [1],
+      "description": "The version of the object."
+    },
+    "dcterms:creator": {
+      "type": "object",
+      "properties": {
+        "ods:type": {
+          "type": "string",
+          "examples": ["foaf:Person"],
+          "description": "Object containing information on who created the object.",
+          "enum": [
+            "foaf:Person",
+            "foaf:Organisation",
+            "oa:SoftwareAgent"
+          ]
+        },
+        "foaf:name": {
+          "type": "string",
+          "examples": ["My Name"],
+          "description": "Name of the creator."
+        }
+      },
+      "description": "The creator of the collection."
+    },
+    "ltc:ObjectGroup": {
```

Comment:
Should this be `ltc:objectGroup`? With lower case "o"? 
If so, it should also be changed in the "required" fields at the end of the schema.

Reply:
it is with uppercase in ltc: https://tdwg.github.io/ltc/terms/index.html#ObjectGroup_ObjectGroup